build.gradle

```diff
@@ -143,6 +152,40 @@ project('api'){
         archives javadocJar
     }
 
+    publishing {
+        publications {
+            Production(MavenPublication){
+                groupId 'com.empowerops'
+                artifactId volitionName
+                version volitionVersion
```

Comment:
Does this publish for maven?

Reply:
kinda? it publishes to a a bintray staging area.

if you then log in and click "deploy to jcenter", it deploys to a maven server here:

http://jcenter.bintray.com/com/empowerops/volition-api/

but, it still doesnt show up in intelliJ.

Apparently maven central also does include artifacts in jcenter, but I cant find them.

- its deployed to bintray, but thats not a maven server, so you cant get it by maven clients:
   https://bintray.com/search?query=com.empowerops
- maven.org only includes sonatype releases:
   https://search.maven.org/search?q=com.empowerops
- mvnrepository.com says it includes jcenter, but I cant find it: 
   https://mvnrepository.com/search?q=com.empowerops 
   > this might just be a weekly refresh that they do? I'm not sure

---

In any event, you can use the `gradle uploadBintray` task to publish the `api.jar` to a place where it can be downloaded publicly from the internet. To do that you need the empowerops bintray API key, which I cant list here because its public. I'll sync up with you to get you the key so you can publish artifacts as you make changes.